packages/react-examples/src/react-charting/VerticalStackedBarChart/VerticalStackedBarChart.CustomAccessibility.Example.tsx

```diff
@@ -134,15 +134,23 @@ export class VerticalStackedBarChartCustomAccessibilityExample extends React.Com
             { y: 10, legend: 'Recommended Builds', color: DefaultPalette.redDark },
           ],
         }),
-        stackCallOutAccessibilityData: { ariaLabel: 'Bar stack series 1 of 6' },
+        stackCallOutAccessibilityData: {
+          ariaLabel:
+            'Bar stack series 1 of 6, 0 MetaDate1 61% MetaData2 8% MetaDate3 31% ' +
+            'Recommended Builds 10 Supported Builds 42',
+        },
       },
       {
         chartData: secondChartPoints,
         xAxisPoint: 20,
         ...(showLine && {
           lineData: [{ y: 33, legend: 'Supported Builds', color: DefaultPalette.magentaLight }],
         }),
-        stackCallOutAccessibilityData: { ariaLabel: 'Bar stack series 2 of 6' },
+        stackCallOutAccessibilityData: {
+          ariaLabel:
+            'Bar stack series 2 of 6, 0 MetaDate1 61% MetaData2 8% MetaDate3 31% ' +
+            'Recommended Builds 10 Supported Builds 42',
```

Comment:
This aria label is incorrect.
Bar stack series 2 of 6, MetaDate1 33% MetaData2 22% MetaDate3 45% ' +
            'Supported Builds 33'